source-control/git.md

```diff
@@ -73,7 +95,8 @@ This section is merely a tl;dr summary.
 7. Use the body to explain what and why vs. how
 
 Here an example:
-```
+
+```md
```

Comment:
md formatting on this block doesn't look ok:
![image](https://user-images.githubusercontent.com/1629215/74343372-a02cd980-4db3-11ea-9f0d-5897c58eccac.png)

I suggest removing.

Reply:
Fixed